test/ex_doc/language/elixir_test.exs

```diff
@@ -171,7 +171,7 @@ defmodule ExDoc.Language.ElixirTest do
 
     test "erlang type" do
       assert autolink_doc("`t::array.array/0`") ==
-               ~s|<a href="https://www.erlang.org/doc/man/array.html#type-array"><code class="inline">:array.array/0</code></a>|
+               ~s|<a href="https://www.erlang.org/doc/man/array.html#t:array/0"><code class="inline">:array.array/0</code></a>|
```

Comment:
✅ https://www.erlang.org/doc/man/array.html#t:array/0 redirects to https://www.erlang.org/doc/apps/stdlib/array.html#t:array/0